myserver.cpp

```diff
@@ -0,0 +1,626 @@
+#include "myserver.h"
+#include "IDataBase.cpp"
+#include "IDataBase.h"
+#include "DBErrors.h"
+#include <iostream>
+#include <vector>
+#include <string>
```

Comment:
myserver.h has this includes. Do you really need them here

Reply:
Fixed it